awsx/schema.json

```diff
@@ -623,15 +611,15 @@
\t\t\t},\n\t\t\tGatewayLoadBalancerArns: pulumi.StringArray{\n\t\t\t\taws_lb.Example.Arn,\n\t\t\t},\n\t\t})\n\t\tif err != nil {\n\t\t\treturn err\n\t\t}\n\t\t_, err = ec2.NewVpcEndpoint(ctx, \"exampleVpcEndpoint\", \u0026ec2.VpcEndpointArgs{\n\t\t\tServiceName: exampleVpcEndpointService.ServiceName,\n\t\t\tSubnetIds: pulumi.StringArray{\n\t\t\t\taws_subnet.Example.Id,\n\t\t\t},\n\t\t\tVpcEndpointType: exampleVpcEndpointService.ServiceType,\n\t\t\tVpcId:           pulumi.Any(aws_vpc.Example.Id),\n\t\t})\n\t\tif err != nil {\n\t\t\treturn err\n\t\t}\n\t\treturn nil\n\t})\n}\n```\n```java\npackage generated_program;\n\nimport com.pulumi.Context;\nimport com.pulumi.Pulumi;\nimport com.pulumi.core.Output;\nimport com.pulumi.aws.AwsFunctions;\nimport com.pulumi.aws.inputs.GetCallerIdentityArgs;\nimport com.pulumi.aws.ec2.VpcEndpointService;\nimport com.pulumi.aws.ec2.VpcEndpointServiceArgs;\nimport com.pulumi.aws.ec2.VpcEndpoint;\nimport com.pulumi.aws.ec2.VpcEndpointArgs;\nimport java.util.List;\nimport java.util.ArrayList;\nimport java.util.Map;\nimport java.io.File;\nimport java.nio.file.Files;\nimport java.nio.file.Paths;\n\npublic class App {\n    public static void main(String[] args) {\n        Pulumi.run(App::stack);\n    }\n\n    public static void stack(Context ctx) {\n        final var current = AwsFunctions.getCallerIdentity();\n\n        var exampleVpcEndpointService = new VpcEndpointService(\"exampleVpcEndpointService\", VpcEndpointServiceArgs.builder()        \n            .acceptanceRequired(false)\n            .allowedPrincipals(current.applyValue(getCallerIdentityResult -\u003e getCallerIdentityResult.arn()))\n            .gatewayLoadBalancerArns(aws_lb.example().arn())\n            .build());\n\n        var exampleVpcEndpoint = new VpcEndpoint(\"exampleVpcEndpoint\", VpcEndpointArgs.builder()        \n            .serviceName(exampleVpcEndpointService.serviceName())\n            .subnetIds(aws_subnet.example().id())\n            .vpcEndpointType(exampleVpcEndpointService.serviceType())\n            .vpcId(aws_vpc.example().id())\n            .build());\n\n    }\n}\n```\n```yaml\nresources:\n  exampleVpcEndpointService:\n    type: aws:ec2:VpcEndpointService\n    properties:\n      acceptanceRequired: false\n      allowedPrincipals:\n        - ${current.arn}\n      gatewayLoadBalancerArns:\n        - ${aws_lb.example.arn}\n  exampleVpcEndpoint:\n    type: aws:ec2:VpcEndpoint\n    properties:\n      serviceName: ${exampleVpcEndpointService.serviceName}\n      subnetIds:\n        - ${aws_subnet.example.id}\n      vpcEndpointType: ${exampleVpcEndpointService.serviceType}\n      vpcId: ${aws_vpc.example.id}\nvariables:\n  current:\n    fn::invoke:\n      Function: aws:getCallerIdentity\n      Arguments: {}\n```\n{{% /example %}}\n{{% /examples %}}\n\n## Import\n\nUsing `pulumi import`, import VPC Endpoints using the VPC endpoint `id`. For example:\n\n```sh\n $ pulumi import aws:ec2/vpcEndpoint:VpcEndpoint endpoint1 vpce-3ecf2a57\n```\n ",
```

Comment:
These diffs are always fun to read.

I've been doing some weekend prototyping on a diff reading tool in https://github.com/t0yv0/pus maybe I can testdrive it here.